code/game/gamemodes/revolution/revolution.dm

```diff
@@ -13,10 +13,10 @@
 	antag_flag = ROLE_REV
 	false_report_weight = 10
 	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer")
-	required_players = 20
-	required_enemies = 1
+	required_players = 30
+	required_enemies = 2
 	recommended_enemies = 3
-	enemy_minimum_age = 14
+	enemy_minimum_age = 18
```

Comment:
Undocumented, and no reason to change it. Teens can be rebellious

Reply:
kek